tests/playwright-tests/admin/azpage/AzLoginPage.ts

```diff
@@ -0,0 +1,35 @@
+import { Locator, Page } from '@playwright/test';
+import  environment  from '../../utils/env';
+
+// AzureLogin Page
```

Comment:
Could remove this comment? The class already says what it is, so this doesn't add anything?

We could get rid of these comments on the other page classes as well!